desc/objectives/_geometry.py

```diff
@@ -613,8 +620,9 @@ def __init__(
         self._surface_grid = surface_grid
         self._plasma_grid = plasma_grid
         self._use_softmin = use_softmin
+        self._use_signed_distance = use_signed_distance
         self._surface_fixed = surface_fixed
-        self._alpha = alpha
+        self._softmin_alpha = softmin_alpha
```

Comment:
should add a deprecation warning for `alpha` if we're changing the name: 
https://github.com/PlasmaControl/DESC/blob/8c4051a555365f87c34871cf073a35f2c0d06e93/desc/utils.py#L492-L493